### DISCUSSION TITLE
ETH Address not matching with OpenSea

### DISCUSSION THREAD

**Discussion by rozztopher** [Web3Auth/Web3Auth#1160] [Help]
Hello, when we use the @web3auth/modal module we can login without issue however, the address we are assigned with the login modal does not match the address that we get on OpenSea when we connect with the Torus wallet option.

Is there any way that we can make the address obtained from the modal match the address on OpenSea so that we have one consistent singular address?

The address I get from our modal is
0x06ec3e1024c9CD06c03d4A2848F17d5BB66cD339

and the address I get from OpenSea is
0x66Dc56F0296Fc19088151D6F859CcE535930a60c

I have added the suggested adapter and plugin from
https://github.com/Web3Auth/web3auth-web/discussions/872

and I've enabled interoperability as suggested here
https://github.com/Web3Auth/web3auth-web/discussions/374

However, I'm still getting the same issue. Is it possible or not to make the address match the one from OpenSea?

**Comment by shahbaz17** [comment]
Hey @rozztopher 

For `@web3auth/modal`, which login provider are you using? 

For this to work, one has to use Torus Wallet from Modal to get the same account as OpenSea (via Torus Wallet)
